aws/aws_instance.go

```diff
@@ -188,9 +188,15 @@ func (awsI *Ec2Instance) runTerraformApply(volumeSize int, zookeeperSize int, ma
 
 	awsI.SdSize = vol.ClusterSize
 	awsI.HTTPMask = mask
-	awsI.RootVolumeType = "standard"
 	awsI.RootVolumeSize = volumeSize
 
+	envVolType := os.Getenv("TF_VAR_root_volume_type")
+	if envVolType != "" && envVolType != "io1" {
+		awsI.RootVolumeIops = 0
+	} else {
+		awsI.RootVolumeIops = volumeSize * 50
+	}
```

Comment:
If the environment have both TF_VAR_root_volume_type=io1 and TF_VAR_root_volume_iops set the value for TF_VAR_root_volume_iops will be ignored and set to *50.  Is that expected?

Reply:
More or less. I figured we always just want the max allowed IOPS and that way the user doesn't have to do the math and try to set it correctly. If you want fewer IOPS, provision a smaller volume.